### PR TITLE
Betrue/fix check

### LINKED ISSUE #2714: Selection check in Chrome is too high.
image

### PR DESCRIPTION
**Pull request by betrue-final-final**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #2714 
- [x] Include a change request file using `$ npm run change`

Switched layout to flex box from transform trick.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Check/Check.scss [hunk 1]
```diff
@@ -58,8 +60,8 @@ $checkBoxHeight: 18px;
   height: auto;
   color: $ms-color-neutralTertiaryAlt;
   font-size: 16px;
-  top: 40%;
-  left: 53%;
+  // top: 40%;
+  // left: 53%;
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Check/Check.scss]: remove these lines

File: common/changes/office-ui-fabric-react/betrue-fixCheck_2017-09-01-18-18.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fixed chrome check placement bug, and converted to flex box",
```
Comment (micahgodbolt) [inline common/changes/office-ui-fabric-react/betrue-fixCheck_2017-09-01-18-18.json]: flex all the things!

**Comment by dzearing** [comment]
We REALLY need dev links for these sorts of things. It would help tremendously with code coverage and gating fixes that only were tested in one browser.

**Comment by Jahnp** [comment]
10000% agreed. Happy to chat offline about plans & timing here—I'd love to help make this happen.

**Pull request merged**